indexer-app/src/cmr/indexer/services/index_service.clj

```diff
@@ -374,6 +406,20 @@
         (index-concept context concept parsed-concept options)
         (log-ingest-to-index-time concept)))))
 
+(defn- cascade-collection-delete
+  "Performs the cascade actions of collection deletion,
+  i.e. propagate collection deletion to granules and variables"
+  [context concept-mapping-types concept-id revision-id]
+  (doseq [index (idx-set/get-granule-index-names-for-collection context concept-id)]
```

Comment:
I wouldn't worry about this too much. Re-indexing variables is not a crucial task and I expect us to re-index variables periodically like we do with collections anyways once the bulk indexing endpoint is in place.